### DISCUSSION TITLE
Still keeping this project going?

### DISCUSSION THREAD

**Discussion by smolinari** [TriPSs/nestjs-query#25] [General]
@TriPSs - Hey. 

Are you still going to keep this project on track/ up to date?

Scott

**Comment by TriPSs** [comment]
Hi Scott, yes! My own applications heavily depend on it and i'm not planning on migrating away.

**Reply by smolinari** [reply]
Are you going to update the dependencies? Or all ok right now? DIdn't check directly, only saw the package hadn't had a new version for 3 months. 

Scott

**Reply by smolinari** [reply]
Also, are you going to get them out of alpha?

Scott

**Reply by TriPSs** [reply]
I think we can remove the alpha tag indeed.

Will check if I can update some deps again this weekend :)

**Reply by smolinari** [reply]
Cool. Thanks. 

Scott
